HeterogeneousCore/CUDAUtilities/test/prefixScan_t.cu

```diff
@@ -136,10 +136,13 @@ int main() {
     cudaCheck(cudaMalloc(&d_pc, sizeof(int32_t)));
     cudaCheck(cudaMemset(d_pc, 0, 4));
```

Comment:
`4` here is the same as `sizeof(int)`, like above ?

Reply:
yes

Reply:
should I change all these magic "4"?

Reply:
If you don't mind, yes, as I think it would help understand what is going on.

In fact, I don't know where `4` should come from a call to `wsSize()`, and where it's an independent use of `int` 🤔

Reply:
ok
for the Histo I'm removing the external allocation (this will remove the  need of memset as well)